File: LLM-Translations/GPT-4/codenet/Python/Go/s208095748.go
package main

import (
	"fmt"
)

func main() {
	var A, B, C int
	fmt.Scanf("%d %d %d\n", &A, &B, &C)

	if B < C {
		fmt.Println(0)
	} else if A*C < B {
		fmt.Println(C)
	} else {
		fmt.Println(B / A)
	}
}
// 

File: LLM-Translations/GPT-4/codenet/Python/C/s208095748.c
#include <stdio.h>

int main() {
    int A, B, C;
    scanf("%d %d %d", &A, &B, &C);

    if (B < C) {
        printf("0\n");
    } else if (A * C < B) {
        printf("%d\n", C);
    } else {
        printf("%d\n", B / A);
    }

    return 0;
}


